react-app-drawer-component/src/app-drawer.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+export const navMenu = ['About', 'Get Started', 'Sign In'];
+
+export class AppDrawer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      menuView: false,
+      navToggle: 'off',
+      overlayToggle: ' none'
```

Comment:
Since these can be decided based on `menuView`, you should avoid storing them in state. Instead, choose them in your `render()` method.